sql/core/src/test/resources/sql-tests/results/ansi/datetime.sql.out

```diff
@@ -1,5 +1,65 @@
 -- Automatically generated by SQLQueryTestSuite
--- Number of queries: 85
+-- Number of queries: 92
+
+
+-- !query
+select TIMESTAMP_SECONDS(1230219000),TIMESTAMP_SECONDS(-1230219000),TIMESTAMP_SECONDS(null)
+-- !query schema
+struct<timestamp_seconds(CAST(1230219000 AS BIGINT)):timestamp,timestamp_seconds(CAST(-1230219000 AS BIGINT)):timestamp,timestamp_seconds(CAST(NULL AS BIGINT)):timestamp>
```

Comment:
@cloud-fan Shall we regenerate it for cast(... as Int) ?

Reply:
we have to, otherwise the test fails